tests/apparmor-profiles/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check apparmor profile generation for apps and hooks.
+prepare: |
+  for snap in basic-binaries basic-hooks
+  do
+    snapbuild ./../fixtures/snaps/$snap .
+  done
+restore: |
+  for snap in basic-binaries basic-hooks
+  do
+    sudo snap remove $snap
+    rm ./${snap}_1.0_all.snap
+  done
+execute: |
+  echo "AppArmor profiles are generated and loaded for apps"
+  sudo snap install ./basic-binaries_1.0_all.snap
```

Comment:
Spread runs everything as root by default, so sudo may be dropped.

I'd also suggest dropping all the "./" prefixes.